set.mm

```diff
@@ -748370,9 +748375,9 @@ be a Fermat number (i.e., a Fermat prime), see ~ 2pwp1prmfmtno .
 
   ${
     $d K m n p $.  $d k m p $.
-    $( For every prime number of the form ` ( ( 2 ^ k ) + 1 ) ` ` k ` must be a
-       power of 2, see Wikipedia "Fermat number", section "Other theorms about
-       Fermat numbers", ~ https://en.wikipedia.org/wiki/Fermat_number ,
+    $( For the positive integer ` ( ( 2 ^ k ) + 1 ) ` to be prime, ` k ` must
+       be a power of 2, see Wikipedia "Fermat number", section "Other theorms
```

Comment:
```suggestion
    $( If ` ( ( 2 ^ k ) + 1 ) ` is prime for positive integer ` k ` , ` k ` is
       a power of 2, see Wikipedia "Fermat number", section "Other theorms
```

assuming I'm interpreting the intent of this change correctly, a fix for the case where $$2^{\log_2{6}} + 1$$ is technically a prime positive integer could be like this

Reply:
After more thinking, this is probably not the intent. You may optionally exclude ``for positive integer ` k ` `` since that's rather inherent/obvious, or simply remove `the positive integer` from the start (primes are positive integers so it's redundant) instead of doing my suggestion

Reply:
I'll do the latter. My concern was purely English style.